src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/CuPurchaseOrderAmendmentAccountValidation.java

```diff
@@ -0,0 +1,111 @@
+package edu.cornell.kfs.module.purap.document.validation.impl;
+
+import java.util.List;
+
+import org.kuali.kfs.krad.util.GlobalVariables;
+import org.kuali.kfs.module.purap.PurapConstants;
+import org.kuali.kfs.module.purap.PurapKeyConstants;
+import org.kuali.kfs.module.purap.businessobject.PurApAccountingLine;
+import org.kuali.kfs.module.purap.businessobject.PurApItem;
+import org.kuali.kfs.module.purap.document.PurchaseOrderDocument;
+import org.kuali.kfs.module.purap.document.service.PurchaseOrderService;
+import org.kuali.kfs.module.purap.document.validation.impl.PurchaseOrderAmendmentAccountValidation;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.document.validation.event.AttributedDocumentEvent;
+
+public class CuPurchaseOrderAmendmentAccountValidation extends PurchaseOrderAmendmentAccountValidation {
+
+    /**
+     * Overridden to allow POAs to use expired accounts.
+     * 
+     * @see org.kuali.kfs.module.purap.document.validation.impl.PurchaseOrderAmendmentAccountValidation#validate(
+     * org.kuali.kfs.sys.document.validation.event.AttributedDocumentEvent)
+     */
+    @Override
+    public boolean validate(AttributedDocumentEvent event) {
+
+        boolean valid = true;
+        PurchaseOrderDocument poaDocument = (PurchaseOrderDocument) event.getDocument();
+        List<PurApItem> items = poaDocument.getItemsActiveOnly();
+
+        PurchaseOrderDocument po = SpringContext.getBean(PurchaseOrderService.class).getCurrentPurchaseOrder(poaDocument.getPurapDocumentIdentifier());
+        List<PurApItem> poItems = po.getItems();
+
+        for (PurApItem item : items) {
+            String identifierString = item.getItemIdentifierString();
+
+            if (item.getItemTypeCode().equals(PurapConstants.ItemTypeCodes.ITEM_TYPE_ITEM_CODE)
+                    && item.getSourceAccountingLines() != null && item.getSourceAccountingLines().size() > 0) {
+
+                if (isItemChanged(item, poItems)) {
+                    List<PurApAccountingLine> accountingLines = item.getSourceAccountingLines();
+                    for (PurApAccountingLine accountingLine : accountingLines) {
+                        if (!accountingLine.getAccount().isActive()) {
+                            valid = false;
+                            GlobalVariables.getMessageMap().putError(PurapConstants.ITEM_TAB_ERROR_PROPERTY,
+                                    PurapKeyConstants.ERROR_ITEM_ACCOUNT_INACTIVE, accountingLine.getAccount().getAccountNumber());
+                            break;
+                        }
+                    }
+                }
+            }
+        }
+
+        return valid;
+    }
+
+    // Copied and tweaked this superclass method, and increased its visibility.
+    protected boolean isItemChanged(PurApItem poaItem, List<PurApItem> poItems) {

```

Comment:
I am not sure there is a better solution for this. We can put in a request with KualiCo to increase vivibility for this method in baseline so that we can remove it from the customized file.

Reply:
I agree that getting KualiCo to increase the method visibility would be the best option in this case. I will leave this code as-is, then, unless there are other issues discovered with it.